docs/UpdatedData.md

```diff
@@ -268,12 +272,12 @@ The below fields are mandatory if `alarmType` is set to `handset`. They remain o
 
 ## Zen mode , Confidential mode && Sensor Warmup
 
-Leveraging the `deviceEvent` type and creating 2 new subTypes with the same structure: `zen` and `confidential`.
+Leveraging the `deviceEvent` type and creating new subTypes with the same structure: `zen`, `confidential`, `warmup` and `loopMode` .
 
-- `subType`: `zen | confidential | warmup`
-- `duration`: is a structured object that gives the duration of the event. __This field is mandatory__.
+- `subType`: `zen | confidential | warmup | loopMode`
+- `duration`: is a structured object that gives the duration of the event. __This field is mandatory for the subType `zen`, `confidential`, and `warmup`.__ It is optional for loopMode subType (the duration is updated at the end of the event).
 - `eventId`: unique ID provided by the client that is used to link stop and start events. __This ID is mandatory__.
```

Comment:
should we remove the eventId as we have activated the guid for all datum?

Reply:
Old dblXX will keep pushing eventId field, and some front end operations (0 duration physical activities interpreted as deleted ones for example) also use the eventId to filter some data.

Reply:
switch to guid done and implemented here (when **guid** is present and not empty **guid** overrides **eventId**): 
https://github.com/mdblp/platform/pull/104